source/crud/crud.rst

```diff
@@ -1438,35 +1438,47 @@ Any result class with all parameters marked NOT REQUIRED is ultimately NOT REQUI
 
   }
 
-
+~~~~~~~~~~~~~~
```

Comment:
While I was modifying WriteError, I realized that "Error Handling" applies just as much to `findAndModify` as it does to the "Basic" (insert/update/delete) section.